x/auth/ante/ante_test.go

```diff
@@ -38,6 +38,7 @@ func TestSimulateGasCost(t *testing.T) {
 			"tx with 150atom fee",
 			func(suite *AnteTestSuite) TestCaseArgs {
 				accs := suite.CreateTestAccounts(3)
+				suite.circuitBreakerKeeper.EXPECT().IsCircuitOpen(gomock.Any()).Return(false)
```

Comment:
I don't have enough knowledge and context on these tests so I'm not 100% sure if these `EXPECT` calls are necessary. Maybe @julienrbrt can chime in. But hunch is that they should not be needed for tests that aren't explicitly testing circuit breaker logic